src/Helpers/Conversations/TappedViewOffer.ts

```diff
@@ -0,0 +1,28 @@
+import { ActionType, OwnerType } from "../../Schema"
+import { TappedViewOffer } from "../../Schema/Events/Conversations"
+
+export interface TappedViewOfferArgs {
+  id: string
+  cta: string
+}
+
+/**
+ * A user tapped the view offer CTA in an inquiry
+ *
+ * @example
+ * ```
+ * tappedViewOffer({ id: "123", cta: "Offer Accepted" })
+ * ```
+ *
+ */
+export const tappedViewOffer = ({
+  id,
```

Comment:
Non-blocking: `id` may not be clear about what object we are referring to. How about making it explicit with `impulseConversationId`?